website/docusaurus.theme.tsx

```diff
@@ -0,0 +1,309 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import type {ReactNode} from 'react';
+import React from 'react';
+import Link from '@docusaurus/Link';
+import Translate from '@docusaurus/Translate';
+import VersionsArchived from './versionsArchived.json';
+import PrismLight from './src/utils/prismLight';
+import PrismDark from './src/utils/prismDark';
+import type * as Preset from '@docusaurus/preset-classic';
+// TODO remove this eslint ruse
+// TODO PrismDark and PrismLight types
+// TODO  `satisfies Preset.ThemeConfig`  `satisfies Config`
+// results Expected ';', got 'satisfies'
+/* eslint-disable @docusaurus/no-untranslated-text */
+const ArchivedVersionsDropdownItems = Object.entries(VersionsArchived).splice(
+  0,
+  5,
+);
+const isDev = process.env.NODE_ENV === 'development';
+
+export default {
+  announcementBar: {
```

Comment:
process is not defined for agolia however no error reported for navbar Debug label with isDev